### PR TITLE
[12.0][MIG] account_move_so_import

### PR DESCRIPTION
**Pull request by i-vyshnevska**
base work done in https://github.com/OCA/account-reconcile/pull/270

### CODE REVIEW

**Review by grindtildeath — APPROVED** [review]
LGTM

File: account_move_so_import/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    'maintainer': 'Camptocamp',
+    'category': 'Finance',
+    'complexity': 'easy',
+    'depends': ['account_move_base_import', 'sale'],
+    'website': 'http://www.camptocamp.com',
+    'data': [
+        'data/completion_rule_data.xml',
+    ],
+    'installable': True,
+    'images': [],
```
Comment (TDu) [inline account_move_so_import/__manifest__.py]: I think empty keys should be removed.

**Review by sebalix — APPROVED** [review]

**Review by TDu — APPROVED** [review]

**Review by yvaucher — APPROVED** [review]

**Comment by yvaucher** [comment]
@i-vyshnevska can you rebase to launch travis

https://github.com/OCA/account-reconcile/pull/270 is now merged, thus the module will now be available in tests.

**Pull request closed without merging**